tox.ini

```diff
@@ -18,6 +18,7 @@ commands = flake8 {posargs:.}
 deps = dennis==0.9
 # Run via `bash -c` to make sure the file globbing is properly resolved in the shell
 commands =
+  bash -c 'dennis-cmd lint locale/*/LC_MESSAGES/*.po --rules W202'
```

Comment:
Huh. Should we turn on more rules?

Reply:
I mean, are there other problems we might be missing?

Reply:
@kumar303 Here's the table of possible errors and warnings https://dennis.readthedocs.io/en/latest/linting.html#table-of-warnings-and-errors. All the errors are covered by --errorsonly but we could add any other warning codes if we thing they're likely to cause problems.